samples/snippets/src/test/java/aiplatform/GetModelEvaluationImageClassificationSampleTest.java

```diff
@@ -33,8 +33,8 @@
 public class GetModelEvaluationImageClassificationSampleTest {
 
   private static final String PROJECT = System.getenv("UCAIP_PROJECT_ID");
-  private static final String MODEL_ID = System.getenv("IMAGE_CLASS_MODEL_ID");
-  private static final String EVALUATION_ID = System.getenv("IMAGE_CLASS_EVALUATION_ID");
+  private static final String MODEL_ID = "3512561418744365056";
+  private static final String EVALUATION_ID = "9035588644970168320";
```

Comment:
similar to `MODEL_ID`, I think we should continue using env var and just update the value in secret manager.

Reply:
Acknowledged.